pyston/filters/managers.py

```diff
@@ -102,6 +106,8 @@ def _get_model_filter(self, identifiers_prefix, identifiers, model, resource, re
         current_identifier = identifiers[0]
         identifiers_suffix = identifiers[1:]
 
+        current_identifier = self._get_real_field_name(resource, current_identifier)
```

Comment:
The `current_identifier` definition few lines above makes now no sense, since it is not used until current line. I suggest putting this to the line I've marked for deletion.
```suggestion
        current_identifier = self._get_real_field_name(resource, identifiers[0])
```

Reply:
yes you are right